src/components/EntityInfo.vue

```diff
@@ -360,20 +360,6 @@ export default class EntityInfo extends Mixins(AllowableActionsMixin, CommonMixi
   // }
 }
 
-.v-breadcrumbs li {
-  font-size: $px-12;
-}
-
-::v-deep {
-  .v-breadcrumbs a {
-    color: $gray8 !important;
-  }
-
-  .v-breadcrumbs a:hover {
-    color: $BCgovABlue3 !important;
-  }
-}
```

Comment:
The breadcrumbs are no longer in this component. (They are their own shared component, rendered in App.vue.)